lib/raven/base.rb

```diff
@@ -227,22 +225,22 @@ def inject_only(*only_integrations)
     def load_integration(integration)
       require "raven/integrations/#{integration}"
     rescue Exception => error
-      self.logger.warn "Unable to load raven/integrations/#{integration}: #{error}"
+      logger.warn "Unable to load raven/integrations/#{integration}: #{error}"
     end
 
     # For cross-language compat
-    alias :captureException :capture_exception
-    alias :captureMessage :capture_message
-    alias :annotateException :annotate_exception
-    alias :annotate :annotate_exception
+    alias_method :captureException, :capture_exception
+    alias_method :captureMessage, :capture_message
+    alias_method :annotateException, :annotate_exception
+    alias_method :annotate, :annotate_exception
 
     private
 
     def install_at_exit_hook(options)
       at_exit do
-        if $!
-          logger.debug "Caught a post-mortem exception: #{$!.inspect}"
-          capture_exception($!, options)
+        if (exc = $!) # rubocop:disable Style/SpecialGlobalVars
```

Comment:
You always have the option of including [English.rb](https://github.com/ruby/ruby/blob/trunk/lib/English.rb#L23) to have `$ERROR_INFO` handy.

That file's been with us a _long_ while.